### ISSUE TITLE
service core:user:elysia-test: Uncaught ReferenceError: process is not defined   at index.js:7397:27

### ISSUE THREAD

**Issue by ryoppippi** [elysiajs/elysia#69]
Related to: #58 
I tried to run elysia on wrangler(cloudflare workers), and I got this error.
The code here below 
```ts
import { Elysia, t } from "elysia";
// import { swagger } from "@elysiajs/swagger";

const app = new Elysia()
  // .use(swagger())
  .get("/", () => "Hello Elysia")
  .get("/id/:id", (context) => context.params.id)
  .post(
    "/post",
    async (event) => {
      const body = event.body;
      return body;
    },
    {
      body: t.Object({
        username: t.String(),
        password: t.String(),
      }),
    }
  );

export default app;
```
`package.json`
```json
{
  "name": "hi-elysia",
  "version": "1.0.44",
  "scripts": {
    "dev": "wrangler dev src/index.ts"
  },
  "devDependencies": {
    "elysia": "0.6.0-alpha.2",
    "superjson": "^1.13.1",
    "tsx": "^3.12.7",
    "wrangler": "^3.3.0"
  },
  "module": "src/index.js"
}
```

The error message
```sh
⎔ Starting local server...
service core:user:elysia-test: Uncaught ReferenceError: process is not defined
  at index.js:7397:27
```

**Comment by ryoppippi** [comment]
Oh, I'm sorry this is related to https://github.com/remix-run/remix/issues/1285

**Comment by ryoppippi** [comment]
Is it able to ignore ‘process’ variable if there is no process variable found

**Comment by ryoppippi** [comment]
I used this commit https://github.com/elysiajs/elysia/issues/58#:~:text=solved%20the%20problem.-,b2df605,-Please%20leave%20this
and I found that this error caused in `error.ts`
https://github.com/elysiajs/elysia/blob/b2df6053a9cdc5c0919146cec2485dd5f487bfa7/src/error.ts#L4-L5

**Comment by SaltyAom** [comment]
Is this issue solved with Elysia 0.6.0 stable?

**Comment by SaltyAom** [comment]
As we can now use Elysia on Cloudflare worker since 0.6.1, I assume this problem is solved.

Feels free to reopen if the issue still persists.
